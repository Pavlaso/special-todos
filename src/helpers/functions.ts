export const toUpperCaseFirsrLetter = (item: string) => item[0].toUpperCase() + item.slice(1)
