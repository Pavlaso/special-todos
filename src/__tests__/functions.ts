import { toUpperCaseFirsrLetter } from "../helpers/functions"

describe('toUpperCaseFirsrLetter', () => {
    it('Validate value', () => {
        expect(toUpperCaseFirsrLetter('string')).toBe('String')
        expect(toUpperCaseFirsrLetter('string')).toBe('String')
    })
    it('toUpperCaseFirsrLetter snapshot', () => {
        expect(toUpperCaseFirsrLetter('string')).toMatchSnapshot()
    })
})
