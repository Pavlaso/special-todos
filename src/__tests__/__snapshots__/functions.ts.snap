// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `"String"`;

exports[`toUpperCaseFirsrLetter toUpperCaseFirsrLetter snapshot 1`] = `"String"`;
